### DISCUSSION TITLE
stream load documents transaction commit url path wrong

### DISCUSSION THREAD

**Discussion by zhangkun-pass** [apache/doris#17532] [RFC]
https://doris.apache.org/zh-CN/docs/dev/data-operate/import/import-way/stream-load-manual
when i use doris 1.1.0 transaction commit url path must use this:   /api/{db}/_stream_load_2pc
But the document is:                                                                            /api/{db}/{table}/_stream_load_2pc
Is this a version problem or an error

**Comment by LemonLiTree** [comment]
Adding {table} is only supported after 1.2.1
https://github.com/apache/doris/pull/15248
